migrations/900000000000058_add_default_syncs_for_container_images.up.sql

```diff
@@ -186,4 +186,40 @@ BEGIN
 END
 $$;
 
+--https://www.graphile.org/postgraphile/computed-columns/
+CREATE OR REPLACE FUNCTION mergestat.container_syncs_latest_sync_runs(container_syncs mergestat.CONTAINER_SYNCS)
+RETURNS JSONB
+LANGUAGE PLPGSQL STABLE
+AS $$
+DECLARE
+   response JSONB;
+BEGIN
+    WITH last_completed_syncs AS(
+        SELECT
+            cs.id AS container_sync_id,
+            ci.name AS container_image_name,
+            j.id AS job_id,
+            j.status,
+            j.started_at,
+            j.completed_at,
+            (SELECT COUNT(1) FROM sqlq.job_logs WHERE sqlq.job_logs.job = j.id AND level = 'warn') warning_count,
+            (SELECT COUNT(1) FROM sqlq.job_logs WHERE sqlq.job_logs.job = j.id AND level = 'error') error_count
+        FROM mergestat.container_syncs cs
+        INNER JOIN mergestat.container_sync_executions cse ON cs.id = cse.sync_id
+        INNER JOIN mergestat.container_images ci ON cs.image_id = ci.id
+        INNER JOIN sqlq.jobs j ON cse.job_id = j.id
+        WHERE cs.id = container_syncs.id
+        ORDER BY cs.id, j.completed_at DESC
```

Comment:
@amenowanna Should this commit have the `created_at` here in't it? and I am thinking it is enough just sorting from here by `created_at` and in the fronted it wouldn't be necessary

Reply:
I just remembered `JSONB_OBJECT_AGG` changes the order jeje, so it is still necessary apply the sorting from Frontend.

Reply:
Good catch on both! I i didn't notice any issues with the sort though. I just pushed my fix.